### DISCUSSION TITLE
Como crear proyecto publico en la nube , para agregar colaboradores en la version FREE de qfield.

### DISCUSSION THREAD

**Discussion by cesararevalo0594** [opengisch/QField#5501] [Q&A]
La situacion es la siguiente, ya he elaborado mi proyecto en QGIS, y lo empaqueto para poder subir a la nube. cuando lo hago siempre me da como proyecto propio.
yo quiero que sea publico , para poder agregar colaboradores(alumnos). seria de gran ayuda algun documento o mejor un video tutorail que me muestre todos los pasos para llevar el proyecto a la nube y que los colaboradores pueden ser agrergados. para cuando ellos con su propio usuario y contraseñas puedan ingresar al proyecto en la nube , y no tener que yo enviarles mi usuario y contraseña. 
y otra forma que podria hacer es a traves de un proyecto en la nube vacio, y cargar las capas, ademas de los colaboraores, como se procede para cargar esas capas al proyecto vacio.
espero ansioso vuestra ayuda. muchas gracias

**Comment by SeqLaz** [comment]
Hola @cesararevalo0594 ,

Para cambiar el proyecto a público en QFieldCloud, ingrese al proyecto, vaya a Configuración (Settings) y active la casilla “Es público (Is Plublic)”. Luego, en la sección de Colaboradores (Collaborators), puede buscar las cuentas ya registradas.

En cuanto a cómo agregar capas a un proyecto vacío en la nube, solo necesitas descargar el proyecto vacío creado usando QFieldSync. Asegúrese de recordar la ubicación de la carpeta donde necesitas pegar las capas y/o el proyecto. Asegúrese de agregar las capas al proyecto usando las capas en esta nueva ubicación.

Similar al flujo de trabajo descrito en la segunda opción descrita en la documentación oficial:

https://docs.qfield.org/get-started/tutorials/get-started-qfc/#create-a-project-in-an-organization

![image](https://github.com/user-attachments/assets/ec2c0aaf-89d3-4f63-a765-b4c1d4e01ef2)
